polaris-react/src/components/AppProvider/AppProvider.tsx

```diff
@@ -138,6 +138,8 @@ export class AppProvider extends Component<AppProviderProps, State> {
     const {i18n, linkComponent} = this.props;
 
     this.setRootAttributes();
+    /* Temporary for dynamicTopBarAndReframe feature. Remove when feature flag is removed. */
```

Comment:
Can we track this in a ticket that get's tracked in our project work/team backlog?

Reply:
Started a cleanup task list here! https://github.com/Shopify/polaris-internal/issues/1574